widget_pages/login.py

```diff
@@ -232,6 +260,29 @@ def loginPushed(self):
                 self.parent().parent().is_admin_user = False
             self.showPatientListWindow()
 
+    def loginPageAnimation(self):
```

Comment:
The form moves when the user inputs an incorrect password
https://user-images.githubusercontent.com/44624435/230967971-831a1c48-fa6f-4711-8113-829c9f5a7488.mp4